Entities/PersonalNumber.cs

```diff
@@ -4,14 +4,14 @@
 
 namespace FRITeam.Swapify.Entities
 {
-    public class StudyGroup : BaseEntity
+    public class PersonalNumber : BaseEntity
```

Comment:
Personal number or StudentNumber is the same problem...
What does this class contain? List of courses or timetable is part of the student number? What is the purpose of this class? To store what?

Reply:
Purpose of this class was to be an alternative for group number. This should store timetable for personal number of student. In last modification I remove this class too. Timetable is stored as first attribute in class Student which has personal number as second attribute.

Reply:
I understand you want to save 2 timetables for student: based on his TAPI and changes by student. It is OK, Just the name was wrong (may be the place). If it contains just number (like now) the name is OK...